### PR TITLE
Burial 94168 remove structuredata processdatajob

### PR DESCRIPTION
**Pull request by wayne-weibel**
## Summary

Remove StructuredData::ProcessDataJob

## Related issue(s)

[Burial | Remove StructuredData::ProcessDataJob](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94168)

## Testing done

- [x] *New code is covered by unit tests*
- local submission of burial claim
- local run of related sidekiq job

## What areas of the site does it impact?

Burial

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: lib/burials/monitor.rb [hunk 1]
```diff
@@ -99,5 +99,21 @@ def track_create_success(in_progress_form, claim, current_user)
...
+    # log process_attachments! error
+    # @see BurialClaimsController
+    #
+    # @param in_progress_form [InProgressForm]
+    # @param claim [SavedClaim::Burial]
+    # @param current_user [User]
+    #
+    def track_process_attachment_error(in_progress_form, claim, current_user)
+      StatsD.increment("#{CLAIM_STATS_KEY}.process_attachment_error")
+      Rails.logger.error('21P-527EZ process attachment error',
```
Comment (balexandr) [inline lib/burials/monitor.rb:113]: @wayne-weibel `21P-530EZ`?

**Review by balexandr — APPROVED** [review]
LGTM

File: app/controllers/v0/burial_claims_controller.rb [hunk 2]
```diff
@@ -7,7 +7,18 @@ module V0
...
 
+    # an identifier that matches the parameter that the form will be set as in the JSON submission.
+    def short_name
+      'burial_claim'
+    end
+
+    # a subclass of SavedClaim, runs json-schema validations and performs any storage and attachment processing
+    def claim_class
+      SavedClaim::Burial
+    end
```
Comment (stevenjcumming) [inline app/controllers/v0/burial_claims_controller.rb:18]: Can these be moved to the private scope?

File: lib/burials/monitor.rb [hunk 3]
```diff
@@ -99,5 +99,21 @@ def track_create_success(in_progress_form, claim, current_user)
...
+    # @param claim [SavedClaim::Burial]
+    # @param current_user [User]
+    #
+    def track_process_attachment_error(in_progress_form, claim, current_user)
+      StatsD.increment("#{CLAIM_STATS_KEY}.process_attachment_error")
+      Rails.logger.error('21P-530EZ process attachment error',
+                         { confirmation_number: claim&.confirmation_number, user_uuid: current_user&.uuid,
+                           in_progress_form_id: in_progress_form&.id, errors: claim&.errors&.errors,
+                           statsd: "#{CLAIM_STATS_KEY}.process_attachment_error" })
+    end
```
Comment (stevenjcumming) [inline lib/burials/monitor.rb:117]: Please consider reformatting for readability 

```suggestion
    def track_process_attachment_error(in_progress_form, claim, current_user)
      StatsD.increment("#{CLAIM_STATS_KEY}.process_attachment_error")
      context = { 
        confirmation_number: claim&.confirmation_number, 
        user_uuid: current_user&.uuid,
        in_progress_form_id: in_progress_form&.id, 
        errors: claim&.errors&.errors,
        statsd: "#{CLAIM_STATS_KEY}.process_attachment_error" 
      }
      Rails.logger.error('21P-530EZ process attachment error', context)
    end
```

**Comment by wayne-weibel** [comment]
@stevenjcumming Would you mind re-reviewing this PR as well

**Review by stevenjcumming — APPROVED** [review]
looks good

**Pull request merged**